test/release/examples/primers/forallLoops.chpl

```diff
@@ -320,31 +326,59 @@ Task-Private Variables
 ----------------------
 
 A task-private variable is similar to an in-intent or ref-intent
-shadow variable in that it is initialized at the beginning of its
-task and deallocated at the end of the task. However, a task-private
-variable is initialized without regard to any outer variable.
+shadow variable in that it is set up at the beginning of its
+task before the task executes any loop iterations, and deallocated
+at the end of the task after the task completes all "its" iterations.
+Each task created by the parallel iterator gets its own set
+of task-private variable(s).
+Task-private variables are different in that they have no corresponding
```

Comment:
How about "task-private variables are different from shadow variables because they do not correspond to outer variables".